ads/README.md

```diff
@@ -46,7 +46,7 @@ More information can be provided in a similar fashion if needed (Please file an
 
 - `window.context.noContentAvailable` is a function that the ad system can call if the ad slot was not filled. The container page will then react by showing fallback content or collapsing the ad if allowed by AMP resizing rules.
 - `window.context.reportRenderedEntityIdentifier` MUST be called by ads, when they know information about which creative was rendered into a particular ad frame and should contain information to allow identifying the creative. Consider including a small string identifying the ad network. This is used by AMP for reporting purposes. The value MUST NOT contain user data or personal identifiable information.
-- `window.context.renderStart` is a function that the ad system can call if the ad start rendering. The ad would then set the visibility of the iframe to visible. The ad type needs to be included in `waitForRenderStart` list in [integration.js](../3p/integration.js) for this function to be available.
+- `window.context.renderStart(opt_data)` is a function that the ad system can call if the ad start rendering. The ad would then set the visibility of the iframe to visible. The ad type needs to be included in `waitForRenderStart` list in [integration.js](../3p/integration.js) for this function to be available. Ad system can use opt_data object {width, height} to send the rendered ad size which AMP will use for resize. Please check Ad resizing part for more information.
```

Comment:
... the returned ad size to request a resize. ...

Also, pls format opt_data into code style, same for {width, height}.